markdown/ch8.markdown

```diff
@@ -165,7 +165,7 @@ Note a few things:
 
 ## Introducing the Toolz streaming library
 
-This chapter's code example is from Matt Rocklin (who else?), in which he creates a Markov model from an entire fly genome in under 5 minutes on a laptop, using just a few lines of code.
+This chapter's code example is from Matt Rocklin, in which he creates a Markov model from an entire fly genome in under 5 minutes on a laptop, using just a few lines of code.
```

Comment:
This sentence doesn't parse correctly.  Perhaps:

In this chapter's code example, contributed by Matt Rocklin, we create a Markov model from an entire fly genome in under...

or

This chapter's code example, contributed by Matt Rocklin, creates a Markov model...

or

This chapter's code example is from Matt Rocklin, who creates a Markov model...

or

In this chapter's code example, contributor Matt Rocklin creates a ...